chain-tx-enclave-next/mls/src/tree.rs

```diff
@@ -689,89 +700,61 @@ impl Tree {
     /// find the overlap parent node, decrypt the secret and implant it
     ///
     /// returns (leaf_parent_hash, commit_secret)
-    fn apply_path_secrets(
+    pub fn apply_path_secrets(
```

Comment:
or `pub (crate)` ? with some many public APIs, it's hard to know which ones are to be called from outside + what checks are expected to be performed beforehand on arguments

Reply:
done